packages/sdk/Dockerfile

```diff
@@ -67,18 +76,21 @@ ENV LANGUAGE en_US:en
 
 COPY entrypoint.sh /usr/local/bin/
 COPY retar /usr/local/bin/
-COPY --from=genext2fs /usr/local/bin/genext2fs /usr/bin/
+COPY --from=genext2fs /usr/local/bin/xgenext2fs /usr/bin/
```

Comment:
why is this not going to `/usr/local/bin`?

Reply:
It will work anyway, but for better semantics and consistency, we could put at `/usr/local/bin`.